spec/lib/cooked_post_processor_spec.rb

```diff
@@ -1597,7 +1597,6 @@
               <div class="onebox video-onebox">
                 <video width="100%" height="100%" controls="">
                   <source src="#{video_upload.url}">
-                  <a href="#{video_upload.url}" rel="nofollow ugc noopener">#{video_upload.url}</a>
```

Comment:
I know this looks shady @martin-brennan but the upload previews still work fine.

Reply:
Hmm

```
  1) Jobs::PullHotlinkedImages#execute replaces encoded image urls
     Failure/Error:
       expect do Jobs::PullHotlinkedImages.new.execute(post_id: post.id) end.to change {
         Upload.count
       }.by(1)
     
       expected `Upload.count` to have changed by 1, but was changed by 0
```

I guess the link does affect uploads somehow.

I'm going to convert this back to a draft. There are more side effects than I had expected.

Reply:
Do we know why it no longer includes the `<a href>` elements?

Reply:
Because I forgot to use html5_fragments in the `lib/oneboxer.rb` too :p 

Fixed. 🤞 TY!